.github/workflows/dco.yml

```diff
@@ -9,6 +9,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: ./
+        with:
+          exempt-authors: |
```

Comment:
isn't this only for `KineticCafe/actions-dco`. does it do anything here on `actions/checkout@v3`?

Reply:
Each `-` represents a new array line in YAML, so step 1 is `uses: actions/checkout@v3`. Step 2 is `uses: ./` with the configuration. I’m adding a space for clarity between the lines.